rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/fragment/CovidPatientRegisterFragment.java

```diff
@@ -42,4 +45,13 @@ protected String getPatientRegistrationForm() {
     protected String getDefaultSortQuery() {
         return Constants.DBConstants.LAST_INTERACTED_WITH + " DESC";
     }
+
+    @Override
+    public void onSyncComplete(FetchStatus fetchStatus) {
+        super.onSyncComplete(fetchStatus);
+        RDTApplication.getInstance()
+                .getContext()
+                .allSharedPreferences()
+                .saveLastSyncDate(System.currentTimeMillis());
```

Comment:
This values is already saved when sync succeeds.

Reply:
I have removed this and checked the sync time it was showing but that is not the correct date and time. Showing around 5 months ago time.